src/common/redux/actions/api_key/removeKey.ts

```diff
@@ -0,0 +1,27 @@
+// ==LICENSE-BEGIN==
+// Copyright 2017 European Digital Reading Lab. All rights reserved.
+// Licensed to the Readium Foundation under one or more contributor license agreements.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file exposed on Github (readium) in the project repository.
+// ==LICENSE-END==
+
+import { Action } from "readium-desktop/common/models/redux";
+import { IApiKey } from "../../states/api_key";
+
+export const ID = "API_KEY_REMOVE";
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+// interface IPayload extends Pick<IAnnotationState, "uuid"> {
+interface IPayload extends IApiKey {
+}
+
+export function build(param: IApiKey):
```

Comment:
I think we just need the api provider name identifier in parameter not the entire IApiKey interface, but why not